cli/command/service_test.go

```diff
@@ -0,0 +1,322 @@
+package command
+
+import (
+	"net/url"
+	"testing"
+
+	"github.com/quintilesims/layer0/common/models"
+	"github.com/urfave/cli"
+)
+
+func TestCreateService(t *testing.T) {
+	base, ctrl := newTestCommand(t)
+	defer ctrl.Finish()
+	serviceCommand := NewServiceCommand(base.Command())
+
+	req := models.CreateServiceRequest{
+		DeployID:       "dpl_id",
+		EnvironmentID:  "env_id",
+		LoadBalancerID: "lb_id",
+		ServiceName:    "svc_name",
+	}
+
+	base.Resolver.EXPECT().
+		Resolve("deploy", "dpl_name").
+		Return([]string{"dpl_id"}, nil)
+
+	base.Resolver.EXPECT().
+		Resolve("environment", "env_name").
+		Return([]string{"env_id"}, nil)
+
+	base.Resolver.EXPECT().
+		Resolve("load_balancer", "lb_name").
+		Return([]string{"lb_id"}, nil)
+
+	base.Client.EXPECT().
+		CreateService(req).
+		Return("job_id", nil)
+
+	base.Client.EXPECT().
+		ReadJob("job_id").
+		Return(&models.Job{
```

Comment:
[nit] In [my PR](https://github.com/quintilesims/layer0/pull/394/files#diff-5de2225c2e27305b9f2011b5ab955858R35), zpatrick had me condense multi-line Returns, in this case, assign the `&models.Job` to a variable and then pass the variable here.

Reply:
I assume for keeping-like-with-like's sake, you would declare this variable above all of the base.yaddayadda?

Reply:
I've been declaring them immediately above the `base.*.EXPECT()`

Reply:
✅